bin/reth-bench/src/bench/new_payload_fcu.rs

```diff
@@ -69,12 +81,19 @@ impl Command {
 
                 let (safe, finalized) = tokio::join!(safe_block_hash, finalized_block_hash,);
 
-                let safe_block_hash = safe.unwrap().expect("finalized block exists").header.hash;
-                let finalized_block_hash =
-                    finalized.unwrap().expect("finalized block exists").header.hash;
+                // Handle safe and finalized block lookups gracefully - they may not exist for early blocks
+                let safe_block_hash = match safe {
+                    Ok(Some(block)) => block.header.hash,
+                    Ok(None) | Err(_) => head_block_hash, // Fallback to head for early blocks or on error
+                };
+                
+                let finalized_block_hash = match finalized {
+                    Ok(Some(block)) => block.header.hash,
+                    Ok(None) | Err(_) => head_block_hash, // Fallback to head for early blocks or on error
+                };
```

Comment:
I'm ok with this, but let's remove comments here that refer to early blocks, because we use `saturating_sub` above, so the worst case is that we are fetching the genesis block